src/MatchesSnapshots.php

```diff
@@ -130,23 +130,25 @@ protected function getFileSnapshotDirectory(): string
      * matched.
      *
      * Override this method it you want to use a different flag or mechanism
-     * than `-d --update-snapshots`.
+     * than `-d --update-snapshots` or `UPDATE_SNAPSHOTS=true` env var.
      */
     protected function shouldUpdateSnapshots(): bool
     {
-        return in_array('--update-snapshots', $_SERVER['argv'], true);
+        return in_array('--update-snapshots', $_SERVER['argv'], true)
+            || getenv('UPDATE_SNAPSHOTS') === 'true';
```

Comment:
@freekmurze Actually no real early return possible here, except
```suggestion
		if(in_array('--update-snapshots', $_SERVER['argv'], true)) {
			return true
		}

        return getenv('UPDATE_SNAPSHOTS') === 'true';
```

Which is imo not really an improvement.

Reply:
To me this is more clear, thanks! I really like short conditionals